### PR TITLE
redefine model output representation to be two columns 

### PR DESCRIPTION
**Pull request by micokoch**
Addressing issue #201

### CODE REVIEW

**Comment by zkamvar** [comment]
Ah crap. This was part of my changes in #197 🙈

**Review by LucieContamin — APPROVED** [review]
Looks good to me! thanks!

**Review by zkamvar — CHANGES_REQUESTED** [review]
One more change from my previous review that I cannot suggest on the PR itself: 

line 81 needs to be changed to reflect that the model output representation is made up of two columns and is followed by the model output value column.

**Comment by micokoch** [comment]
Hi @zkamvar ! Thanks for looking at the page more carefully. I accepted the changes you suggested and edited line 81. I hope that works for you.

File: docs/source/user-guide/model-output.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ Much of the material in this section has been excerpted or adapted from the [hub
 
 _Model outputs are a specially formatted tabular representation of predictions._
 Each row corresponds to a unique prediction, and each column provides information about what is being predicted, its scope, and its value. 
-Per hubverse convention, **there are two groups of columns**[^model-id], each group serving a specific purpose: (1) the **"task ID"** columns provide details about what is being predicted, and (2) the three **"model output representation"** columns specifies the type of prediction, identifying information about that prediction, and the value of the prediction. 
+Per hubverse convention, **there are two groups of columns and one column for model output**[^model-id]. Each group of columns serves a specific purpose: (1) the **"task ID"** columns provide details about what is being predicted, and (2) the two **"model output representation"** columns specify the type of prediction, and identifying information about that prediction. Finally, (3) the **value** column provides the model output of the prediction.
```
Comment (zkamvar) [inline docs/source/user-guide/model-output.md]:
```suggestion
Per hubverse convention, **there are two groups of columns for model output**[^model-id] followed by a single column for values. Each group of columns serves a specific purpose: (1) the **"task ID"** columns provide details about what is being predicted, and (2) the two **"model output representation"** columns specify the type of prediction and identifying information about that prediction. Finally, (3) the **value** column provides the model output of the prediction.  
```

**Review by zkamvar — CHANGES_REQUESTED** [review]
Looks good! One more change for clarity.

File: docs/source/user-guide/model-output.md (hunk 1 above)
Comment (micokoch) [reply docs/source/user-guide/model-output.md]: From my understanding of #201, the column for model output is the column for values (the others are "metadata about the representation of that output"). I phrased it differently in my most recent commit.

**Review by zkamvar — APPROVED** [review]
LGTM! Thank you for your patience!

**Pull request merged**